cypress/support/Train.js

```diff
@@ -261,7 +261,7 @@ export function BranchChatTurn(originalMessage, newMessage, originalIndex = 0) {
 }
 
 export function SelectAndVerifyEachChatTurn(index = 0) {
-  if (index == 0) editDialogModal.GetAllChatTurns()
+  if (index == 0) editDialogModal.CreateAliasForAllChatTurns()
```

Comment:
Optional:  generally I'm not a fan of one line "if" statements.  Our style is to { } everything

Reply:
Fixed